script/app.js

```diff
@@ -58,28 +60,108 @@ class MyBook {
   };
 }
 
-document.addEventListener('DOMContentLoaded', () => {
-  const newBook = document.querySelector('#newBook');
-  newBook.addEventListener('submit', (event) => {
-    event.preventDefault();
-    const title = event.target.elements[0].value;
-    const author = event.target.elements[1].value;
-    const myNewBook = new MyBook(title, author);
-    myNewBook.addBookToDom(); // append book to the DOM
-    event.target.elements[0].value = '';
-    event.target.elements[1].value = '';
+const updateBookList = () => {
+  const bookData = JSON.parse(localStorage.getItem('bookCollection'));
+  bookData.forEach((bookData) => {
+    const newBook = new MyBook(bookData.title, bookData.author, bookData.id);
+    newBook.addBookToDom(); // append book to the DOM
   });
+};
+
+const addNewBookEvent = () => {
+  try {
+    const newBook = document.querySelector('#newBook');
+    newBook.addEventListener('submit', (event) => {
+      event.preventDefault();
+      const title = event.target.elements[0].value;
+      const author = event.target.elements[1].value;
+      const myNewBook = new MyBook(title, author);
+      MyBook.unRender.push(myNewBook);
+      event.target.elements[0].value = '';
+      event.target.elements[1].value = '';
+    });
+  } catch (e) {
+    console.log(e);
+  }
+};
+document.addEventListener('DOMContentLoaded', () => {
   if (!localStorage.getItem('bookCollection')) {
     try {
       localStorage.setItem('bookCollection', JSON.stringify([]));
     } catch (exc) {
       console.log('failed to create local storage');
```

Comment:
- [ ] It would be nice to remove `console.log` statements from code. While `console.log()` statements is a good debugging technique, it is bad practice to leave them as code after debugging. 👍